modules/exploits/multi/misc/claymore_dual_miner_remote_manager_rce.rb

```diff
@@ -0,0 +1,137 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/exploit/powershell'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::Powershell
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'            => 'Nanopool Claymore Dual Miner APIs RCE',
+      'Description'     => %q{
+        This module takes advantage of miner remote manager APIs to exploit an RCE vulnerability.
+      },
+      'Author'          =>
+        [
+          'reversebrain@snado', # Vulnerability reporter
+          'phra@snado'          # Metasploit module
+        ],
+      'License'         => MSF_LICENSE,
+      'References'      =>
+        [
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2018-1000049'],
```

Comment:
The `nvd.dist.gov` URL can be removed. It is made redundant by providing the `CVE` key.

Please add these instead:

```ruby
['EDB', '44638'],
['URL', 'https://reversebrain.github.io/2018/02/01/Claymore-Dual-Miner-Remote-Code-Execution/'],
```